spacewalk/admin/mgr-monitoring-ctl

```diff
@@ -2,7 +2,22 @@
 if [[ $1 == "status" || $1 == "enable" || $1 == "disable" ]]; then
 	[[ -z $2 ]] && PILLAR="" || PILLAR="pillar=$2"
 	/usr/bin/salt-call --out=json --log-level=error --local --file-root=/usr/share/susemanager/salt state.apply srvmonitoring.$1 $PILLAR
+elif  [[ $1 == "--help" ]]; then
+  echo "Usage: ${0} {enable|status|disable} [pillar_data]"
+  echo ""
+  echo "Actions:"
+  echo "    enable   Enable server monitoring"
+  echo "    disable  Disable server monitoring"
+  echo "    status   Show status of monitored server components"
+  echo ""
+  echo "To enable PostgreSQL monitoring additional connection details have to be provided by setting `pillar_data`."
```

Comment:
Backquotes are not good as they indicate the enclosed command should be executed. Let's try with single quotes. Sorry.

```suggestion
  echo "To enable PostgreSQL monitoring additional connection details have to be provided by setting 'pillar_data'."
```